{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/public/views.py

```diff
@@ -22,6 +22,7 @@ def load_user(user_id):
 def home():
     """Home page."""
     form = LoginForm(request.form)
+    current_app.logger.info('Hello from the home page!')
```

Comment:
Did you mean to leave this in here?

Reply:
I had added this in as an example of using a logger. Given that a user will need to modify the root route for their application anyway, I'll leave this as-is. Thanks for the review!